app/workers/copay_notifications/new_statement_notification_job.rb

```diff
@@ -22,41 +23,11 @@ class NewStatementNotificationJob
 
     def perform(statement)
       StatsD.increment("#{STATSD_KEY_PREFIX}.total")
-      mpi_response = get_mpi_profile(identifier: statement['veteranIdentifier'],
-                                     identifier_type: statement['identifierType'],
-                                     facility_id: statement['facilityNum'])
-
-      if mpi_response.ok?
-        StatsD.increment("#{STATSD_KEY_PREFIX}.mpi.success")
-        create_notification_email_job(vet360_id: mpi_response.profile.vet360_id, icn: mpi_response.profile.icn)
-      else
-        StatsD.increment("#{STATSD_KEY_PREFIX}.mpi.failure")
-        raise mpi_response.error
-      end
-    end
-
-    def create_notification_email_job(vet360_id:, icn:)
-      if vet360_id
-        CopayNotifications::McpNotificationEmailJob.perform_async(vet360_id,
-                                                                  MCP_NOTIFICATION_TEMPLATE)
-      else
-        StatsD.increment("#{STATSD_KEY_PREFIX}.mpi.vet360_not_found")
-        log_exception_to_sentry(CopayNotifications::Vet360IdNotFound.new(icn), {},
-                                { error: :new_statement_notification_job_error })
-      end
-    end
-
-    def get_mpi_profile(identifier:, identifier_type:, facility_id:)
-      if identifier_type == 'edipi'
-        StatsD.increment("#{STATSD_KEY_PREFIX}.edipi")
-        MPI::Service.new.find_profile_by_edipi(edipi: identifier)
-      else
-        StatsD.increment("#{STATSD_KEY_PREFIX}.vista")
-        MPI::Service.new.find_profile_by_facility(
-          facility_id:,
-          vista_id: identifier
-        )
-      end
+      statement_service = DebtManagementCenter::StatementIdentifierService.new(statement)
+      email_address = statement_service.derive_email_address
+      DebtManagementCenter::VANotifyEmailJob.perform_async(email_address, MCP_NOTIFICATION_TEMPLATE)
+    rescue DebtManagementCenter::StatementIdentifierService::UnableToSourceEmailForStatement => e
+      log_exception_to_sentry(e, {}, { info: :unable_to_source_email_for_statement })
```

Comment:
```suggestion
      log_exception_to_sentry(e, {}, { info: :unable_to_source_email_for_statement }, 'info')
```
I believe the level should be passed as the fourth argument as well

`def log_exception_to_sentry(exception, extra_context = {}, tags_context = {}, level = 'error')`